domain/lease/state/state.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/juju/juju/core/logger"
 	"github.com/juju/juju/domain"
 	"github.com/juju/juju/internal/database"
-	"github.com/juju/juju/internal/database/txn"
+	databaseerrors "github.com/juju/juju/internal/database"
```

Comment:
Perhaps `internal/database/errors.go` should be moved to `internal/database/errors/errors.go`?

Reply:
Maybe, it seems to be the common pattern. Let me think about it and I'll land another PR with the changes after this one if required.